src/APIFunctions/User.js

```diff
@@ -249,3 +250,31 @@ export async function checkIfUserExists(email) {
   });
   return status;
 }
+
+/**
+ * This function takes in a list of current users and returns a
+ * filtered user list that is determined by the filter id.
+ * @param {array} users array of all registered users
+ * @param {integer} filterID represents what to filter email by
+ * @returns {array} filtered array of users
+ */
+export function filterUsers(users, filterID) {
```

Comment:
in this function, get rid of traling comments and add a space between the `if/else` blocks and brackets

Reply:
removed trailing comments and added space